test/support/fixtures/xdr.ex

```diff
@@ -30,4 +31,14 @@ defmodule Stellar.Test.Fixtures.XDR do
 
   # set_trustline
   defdelegate set_trustline_flags(trustor, asset, clear_flags, set_flags), to: Trustline
+
+  # predicates
+  defdelegate predicate_unconditional(predicate), to: Predicate
+  defdelegate predicate_and(value, type), to: Predicate
```

Comment:
@einerzg Let's use `Predicates` (plural) for the module name since it is performing multiple actions over a single XDR construction